Docs/index.md

```diff
@@ -0,0 +1,44 @@
+# GitVersion Docs
+GitVersion is a tool to help you achieve Semantic Versioning on your project.
+
+This influences many of the decisions GitVersion has made, please read and understand this page as it will help you start using GitVersion effectively!
+
+## Assumptions/Rules
+### You tag releases when you release, not before
+GitVersion assumes that tags are used to *tag a release* and not used to build a release.
+
+This means that the version is calculated pre-emptively, if you currently tag, build then release that tag then GitVersion will probably not work for you.
```

Comment:
Actually, yeah, this is exactly what we do on ChocolateyGUI as well with GtiVersion, and this works fine for me currently, on beta 2.